src/include/dns-server/dialog-main.rb

```diff
@@ -40,6 +40,9 @@ def initialize_dns_server_dialog_main(include_target)
       @initial_screen = "start_up"
 
       @service = SystemdService.find("named")
+
+      raise "Service 'named' not found. Please, install the required packages." unless @service
+
```

Comment:
Looks OK to me, the client starts the workflow only when the packages are installed, this is just a safety check. This would raise only when the packages are installed but the service is still missing. That would detect some `bind` changes or system inconsistencies...

But the message could be probably improved, the client already checks for installed packages, asking the user for installing the packages again looks weird and a bit confusing...